buildroot-external/patches/homematic/0014-WebUI-ProgramListSort/homematic/WebUI/www/rega/pages/tabs/admin/views/programlist.htm

```diff
@@ -102,100 +108,187 @@
             }
           }
         }
+      }
 
-        if(oUser)
-        {
-          if (oUser.UserTempViewIDs().Count() == 0)
+       if (( bIsFiltered == true) || ( sEnable == "1" ))
+       {
+           string sPID;
+           foreach( sPID, sEnumPrgIDs )
+           {
+           object o = dom.GetObject( sPID );

```

Comment:
Auch in diesem ganzen Absatz passen die Einrückungen nicht. Bitte korrigieren (nach jedem { zwei spaces auf der nächsten zeile).